lib/attrsets.nix

```
@@ -494,8 +494,9 @@ rec {
        zipAttrsWith (name: values: values) [{a = "x";} {a = "y"; b = "z";}]
        => { a = ["x" "y"]; b = ["z"] }
   */
-  zipAttrsWith =
-    builtins.zipAttrsWith or (f: sets: zipAttrsWithNames (concatMap attrNames sets) f sets);
+
+  inherit (builtins) zipAttrsWith;
```

```suggestion
  inherit (builtins) zipAttrsWith;
```
not that people get confused if something belonged to that comment and was removed by accident